srcjs/init.js

```diff
@@ -75,3 +75,8 @@ $( document ).ready(function() {
 export function getAppInstance() {
   return app;
 }
+
+// attach Framework7 to the window object
+// allows usage in browser at later stage (e.g. for offline PWA page)
+window.Framework7 = Framework7;
```

Comment:
@DivadNojnarg, as said, I find this ugly 😿. But can't get it to work with export...